web/themes/custom/pineapple/components/_card.twig

```diff
@@ -0,0 +1,15 @@
+{% set card_type = card_type|default('article') %}
+{% set attributes = attributes|default(create_attribute()).addClass([
+  'card-container',
+]) %}
+<div {{ attributes }}>
+  <div class="card-image-container">
+    <img src="{{ image_url }}" alt="{{ image_alt_text }}">
+  </div>
+  <div class="card-content u-o-3">
+    <h2 class="page-title"><a href="{{ url }}" rel="bookmark">{{ title }}</a></h2>
+      <p>{{ date_posted }}</p>
+      <p>{{ card_body }}</p>
+    </div>
+  </div>
+</div>
```

Comment:
Remove spare closing div tag and indent code appropriately.

Reply:
done